lib/npc-generation/breakGenderNorms.ts

```diff
@@ -0,0 +1,84 @@
+import { random } from '../src/random'
+import { clamp } from '../src/utils'
+import { Town } from '../town/_common'
+import { GenderName } from './raceTraits'
+import { NPC } from './_common'
+
+/**
+ * town.roll.equality,
+ * chance of breaking gender norm
+ */
+const genderEqualityLikelihood: [number, number, GenderName, GenderName][] = [
+  [139, 10, 'woman', 'man'],
+  [99, 10, 'woman', 'man'],
+  [90, 15, 'woman', 'man'],
+  [80, 20, 'woman', 'man'],
+  [70, 30, 'woman', 'man'],
+  [60, 40, 'woman', 'man'],
+  [55, 60, 'woman', 'man'],
+  [50, 100, 'man', 'woman'],
+  [45, 60, 'man', 'woman'],
+  [40, 50, 'man', 'woman'],
+  [30, 40, 'man', 'woman'],
+  [20, 30, 'man', 'woman'],
+  [10, 20, 'man', 'woman'],
+  [5, 15, 'man', 'woman'],
+  [-101, 10, 'man', 'woman']
+]
+
+export function breakGenderNorms (town: Town) {
+  clamp(town.roll.equality, 0, 100)
+
+  // MIGRATION: Not available outside src.
+  /* if (settings.ignoreGender) {
+    return true
+  } */
```

Comment:
Not quite sure what to do about this. If we run this outside of Twine/SugarCube, we don't really have a settings object to refer to.

Reply:
If we have a corresponding boolean property on the `town` object, then we could reference that instead?

Reply:
I suppose, though I dislike having to pass that around.

Reply:
Well, I guess we're already passing it, so it should be fine to to it.